tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>What's the reason for uninstalling and installing <code>tensorflow</code>  again?</p><p> </p><p>I've been told to also not include commands to install <code>tensorflow</code>  given that it's pre-installed in Colab</p><p>and <code>tensorflow</code>  installation can be assumed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>